hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -515,6 +557,7 @@ private long calculateMaxSteps(BalancerClusterState cluster) {
    * approach the optimal state given enough steps.
    */
   @Override
+  @SuppressWarnings("checkstyle:MethodLength")
```

Comment:
Open to the idea that we should actually clean this up a bit instead of suppressing this warning, but the diff here is already big so I didn't want to add a bunch of refactoring that made it harder to read

Reply:
Don't suppress the warning. Just leave it there to continue yelling at us. Static analysis doesn't block the build, only warns.